sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/templating/JavaParserTemplateProcessor.java

```diff
@@ -418,7 +413,8 @@ private boolean addRequestBody(BlockStmt body, HttpRequestContext method) {
         boolean serializationFormatSet;
         if (requestBody != null) {
             serializationFormatSet = configureBodyWithContentType(body, requestBody.getParameterType(),
-                requestBody.getContentType(), requestBody.getParameterName(), isContentTypeSetInHeaders);
+                requestBody.getContentType(), requestBody.getParameterName(), isContentTypeSetInHeaders,
+                processingEnv.getElementUtils(), processingEnv.getTypeUtils());
```

Comment:
Do we need to break down `ProcessingEnvironment` to these properties on it and pass around two parameters rather than one?